test/reducers/datepicker.js

```diff
@@ -3,40 +3,106 @@
 import test from 'ava';
 import sinon from 'sinon';
 import reducer, { initialState } from '../../app/src/js/reducers/datepicker';
+import { allDateRanges } from '../../app/src/js/utils/datepicker';
 import {
   DATEPICKER_DATECHANGE,
   DATEPICKER_DROPDOWN_FILTER,
   DATEPICKER_HOUR_FORMAT
 } from '../../app/src/js/actions/types';
 
+const secondsPerDay = 60 * 60 * 24;
```

Comment:
I thought about that, and didn't because this is a test.  Since this is basically a constant, having it here will catch any accidental change in the other value.  Meaning if someone thought secondsPerDay should be 60*60*24*1000 or something in the other file.

Reply:
But on cleanup, I did it anyway.